cmd/template/framework/files/routes/chi.go.tmpl

```
@@ -42,3 +49,30 @@ func (s *Server) healthHandler(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(jsonResp)
 }
 {{end}}
+
+{{if .AdvancedOptions.Websocket}}
+func (s *Server) websocketHandler(w http.ResponseWriter, r *http.Request) {
+	socket, err := websocket.Accept(w, r, nil)
+
+	if err != nil {
+		log.Print("could not open websocket")
```

We need to surface the error to understand what the error is